mappers/bluetooth_mapper/controller/controller.go

```diff
@@ -20,8 +20,9 @@ import (
 	"encoding/json"
 	"strings"
 
+	"k8s.io/klog"
+
```

Comment:
Do we need a seperation between this and the other vendor imports ?

Reply:
done